README.md

```diff
@@ -51,14 +51,41 @@ Below are some of the settings you may want to use. These should be defined in y
   A list of Django models which will be ignored by Django Easy Audit.
   Use it to prevent logging one or more of your project's models.
   List items can be classes or strings with `app_name.model_name` format.
-
+  
 * `DJANGO_EASY_AUDIT_UNREGISTERED_URLS_EXTRA`
 
   A list of URLs which will be ignored by Django Easy Audit.
   List items are expected to be regular expressions that
   will be matched against the URL path.
   [Check our wiki](https://github.com/soynatan/django-easy-audit/wiki/Settings#request-auditing)
   for more details on how to use it.
+  
+* `DJANGO_EASY_AUDIT_LOGGING_BACKEND`
+
+  A pluggable backend option for logging. Defaults to `easyaudit.backends.ModelBackend`.
+  This class expects to have 3 methods: 
+  * `login(self, login_info_dict):`  
+  * `crud(self, crud_info_dict):`  
+  * `request(self, request_info_dict):`
+  
+  each of these methods accept a dictionary containing the info regarding the event.  
+  example overriding:
+  ```python
+    import logging
+    
+    class PythonLoggerBackend:
+        logger = logging.getLogger('your-kibana-logger')
+        
+        def request(self, request_info):
```

Comment:
For consistency (even though it isn't super useful?), we may wish to return the info object back to the caller.

I'm thinking that if we're making this truly pluggable, then the core classes should try to follow a consistent behavior.

With type annotations, we could try to enforce it style-wise, too.

Reply:
@jheld I agree! Can you type your suggestion?